### DISCUSSION TITLE
laser is shadowing

### DISCUSSION THREAD

**Discussion by EmmaKimNZ** [arkypita/LaserGRBL#1734] [Q&A]
Can someone help me please. When I put a picture into GRBL to be engraved, it shows a lot of gray lines going all the way to the right of the screen and it engraves everything twice so you can't see anything clearly.  My daughters GRBL doesn't do it on the same settings (gray lines just inside the image, not repeating off the side of the screen). I've tried uninstalling and reinstalling but it just reinstalls the same thing complete with files and custom buttons.
![2022-02-02](https://user-images.githubusercontent.com/98866567/152096266-1fc8019f-bc1f-4ade-81f3-a5d3c5064beb.png)

**Comment by arkypita** [comment]
Your source image have a black line on the right side, and LaserGRBL is trying to reproduce the image, so...
Crop your image in a graphic editor (like Paint.NET https://www.getpaint.net/download.html)

**Reply by EmmaKimNZ** [reply]
Thanks Arkypita.  There was no visible line but I tried cropping and it fixed the lines problem.  Not sure why every picture I put in is like that but I'll crop them all as close as I can.  It didn't fix the double printing though - any ideas? This pic is from a different source image so it doesn't have the est 2021 in the middle but the rest is the same.  It's hard to tell from the picture but everything is printed twice making it look blurry. I'm not sure if it is the laser or the settings, but it doesn't happen if I do an outline, and it only happened a little bit when I tried lightburn. It doesn't happen with every image, and it's not the same each time - sometimes worse, sometimes better....
![double printing](https://user-images.githubusercontent.com/98866567/152121357-186a565f-8e90-4bd8-8e21-f99ae6d03710.jpg)
Any ideas appreciated

**Comment by arkypita** [comment]
> Thanks Arkypita. There was no visible line but I tried cropping and it fixed the lines problem.

You probably don't see it because it's very small, but it's there. Or you might have some brightness and contrast setting in LaserGRBL that accentuates it. If you want to send the original image to info@lasergrbl.com I will give it a look.

> It didn't fix the double printing though - any ideas? 

From the image I would say a mechanical problem. You must check that your machine has the belt properly tensioned, that there are no mechanical slack in frame and axis and that it does not move, slip, or vibrate while doing the job etc.
